pra_request_tracker/pra_request_tracker/settings_test.py

```diff
@@ -0,0 +1,5 @@
+from pra_request_tracker.settings import *  # noqa: F403
+
+
+MEDIA_ROOT = APPS_DIR / "media"  # noqa: F405
+del DEFAULT_FILE_STORAGE  # noqa: F821
```

Comment:
I have to say the `del` keyword is very infrequently used. Could the be `DEFAULT_FILE_STORAGE=None` or some other alternative?

Reply:
Oh let me try!

Reply:
Doesn't work. Unfortunately if it's defined at all it'll use the defined value.

Reply:
Hmm, I know this may seem weird/extra work but there's a strong general aversion to `del` in cases where it's not necessary. Maybe we could do this?
```suggestion
if "DEFAULT_FILE_STORAGE" in globals():
    del DEFAULT_FILE_STORAGE  # noqa: F821
```
Just to add an extra check so the program doesn't crash if somehow that's not defined